app/routes/persons.js

```diff
@@ -0,0 +1,30 @@
+// const { getPersonByReference, getPersonAndDogsByReference } = require('../repos/people')
+// const { personDto, personAndDogsDto } = require('../dto/person')
+const { personsFilter } = require('../schema/persons/get')
+const { getPersons } = require('../repos/persons')
+const { personDto } = require('../dto/person')
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/persons',
+    options: {
+      validate: {
+        query: personsFilter,
```

Comment:
I would find it more helpful for 'personsFilter' to be called 'personsSchema' or 'personsParamsSchema' since it's a Joi schema, not really a filter. I think this naming was from when you were filtering the params to validate them